pkg/cmd/openfaas_app_test.go

```diff
@@ -17,3 +22,47 @@ func Test_getValuesSuffix_aarch64(t *testing.T) {
 		t.Errorf("suffix, want: %s, got: %s", want, got)
 	}
 }
+
+var (
```

Comment:
These values should be moved inside the scope of the function please don't use the terms "expected', but `want` and `got`.